modules/cnf-topology-aware-lifecycle-manager-troubleshooting.adoc

```diff
@@ -253,13 +253,14 @@ spec:
     clusters:
     - spoke1
     - spoke3
-    clusterSelector:
-    - upgrade2=true
     remediationStrategy:
         canaries:
         - spoke3
         maxConcurrency: 2
         timeout: 240
+    clusterLabelSelectors:
```

Comment:
This issue is about the canary. The selector example should be part of the issue above?